### ISSUE TITLE
googlechrome update check blocked by ryzen-controller

### ISSUE THREAD

**Issue by phanirithvij** [ScoopInstaller/Extras#8054]
The repo https://github.com/ScoopInstaller/UpdateTracker has issues disabled so opening it here.
The extras/googlechrome update check is blocked by `ryzen-controller`.

https://github.com/ScoopInstaller/UpdateTracker/runs/5342710292?check_suite_focus=true

**Comment by issaclin32** [comment]
Fixed. ([Action Log](https://github.com/ScoopInstaller/UpdateTracker/actions/runs/1903904518))

Thank you for reporting!

**Comment by phanirithvij** [comment]
Can issues be enabled?

**Comment by issaclin32** [comment]
> Can issues be enabled?

Issues is now enabled.

**Comment by phanirithvij** [comment]
@issaclin32 https://github.com/ScoopInstaller/UpdateTracker/issues has templates which I think are not applicable for that repo.

**Comment by issaclin32** [comment]
> @issaclin32 https://github.com/ScoopInstaller/UpdateTracker/issues has templates which I think are not applicable for that repo.

updated.
([new issue](https://github.com/ScoopInstaller/UpdateTracker/issues/new/choose))
